diagnostic.md

```diff
@@ -49,22 +51,25 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
-```
+@function tint($color, $percentage) {
+  @return mix($plain-white, $color, $percentage)
+}
+function tint($cornflower, 20%)
+``
 
 ## Mixins
 
 Suppose you want to define a mixin named `row` stored in `./row.scss`. Write the
 code to import the mixin definition in the current module.
 
 ```scss
-// your response here
+@import '.row.scss'
```

Comment:
Correct. Just watch the `.` before the `row.scss`. It is not needed or you have to have `./row.scss`